src/test/java/domain/NameTest.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+class NameTest {
+
+    @DisplayName("자동차 이름은 1자 이상, 5자 이하여야 한다.")
```

Comment:
ParameterizedTest를 적용하면 좋을 것 같아요!

Reply:
```suggestion
    @DisplayName("1자 이상, 5자 이하로 생성할 수 있다.")
    @ParameterizedTest
    @ValueSource(strings = {"쥬", "쥬니", "쥬니헙크1"})
    void test1(String name) {
        assertDoesNotThrow(() -> new Name(name));
    }
```
다른 Test 코드에서도 위와 같이 @ParameterizedTest를 사용하였습니다.